### ISSUE TITLE
SmartCity-Accessibility Server Side #574 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#343]
Build 'SmartCity-Accessibility Server Side' is failing!

Last 50 lines of build :
```
[...truncated 9.13 KB...]
lat is : 32.081387 lng is : 34.768173
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.148 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  ReviewManagerTest.testBackgroundCalls:70 expected:<null> but was:<false>
  UserProfileManagerTest.testBackgroundCalls:59 expected:<null> but was:<false>

Tests run: 65, Failures: 2, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:37 min
[INFO] Finished at: 2017-05-15T12:06:10+00:00
[INFO] Final Memory: 34M/209M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility Server Side/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] 2d3ea614075c8c5336a02e3cb9bbfd8ae097d5b8 - changed null return from Boolean to false instead

- [Alexander Kaplan] 1be6f405086d3a2faa2a816ea3aa33477ba54c3b - changed null return from Boolean to false instead

- [Alexander Kaplan] ca2a4b2f35a0c3f5db8ddbecf109748bb69e341f - removed suppress warning 'unused' that caused a warning because it was

- [Ariel Kolikant] 68b72bc80e8e1f13776e8ec093729f9099e4b1ad - #305 opens page for adding

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/574/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Last 50 lines of build :
```
[...truncated 8.97 KB...]
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.13 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.152 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  ReviewManagerTest.testBackgroundCalls:70 expected:<null> but was:<false>
  UserProfileManagerTest.testBackgroundCalls:59 expected:<null> but was:<false>

Tests run: 65, Failures: 2, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:53 min
[INFO] Finished at: 2017-05-15T12:21:29+00:00
[INFO] Final Memory: 35M/209M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility Server Side/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] 2d3ea614075c8c5336a02e3cb9bbfd8ae097d5b8 - changed null return from Boolean to false instead

- [Alexander Kaplan] 1be6f405086d3a2faa2a816ea3aa33477ba54c3b - changed null return from Boolean to false instead

- [Alexander Kaplan] ca2a4b2f35a0c3f5db8ddbecf109748bb69e341f - removed suppress warning 'unused' that caused a warning because it was

- [Ariel Kolikant] 68b72bc80e8e1f13776e8ec093729f9099e4b1ad - #305 opens page for adding

- [Ariel Kolikant] 304a108fd641c603947a38043845ce754b827fc1 - tells me i have changes



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/575/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Last 50 lines of build :
```
[...truncated 8.66 KB...]
lat is : 32.065163 lng is : 34.771384
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.1 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.111 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  UserProfileManagerTest.testBackgroundCalls:58 expected:<false> but was:<null>

Tests run: 65, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:39 min
[INFO] Finished at: 2017-05-15T12:23:20+00:00
[INFO] Final Memory: 34M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility Server Side/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] 2d3ea614075c8c5336a02e3cb9bbfd8ae097d5b8 - changed null return from Boolean to false instead

- [Alexander Kaplan] 1be6f405086d3a2faa2a816ea3aa33477ba54c3b - changed null return from Boolean to false instead

- [Alexander Kaplan] ca2a4b2f35a0c3f5db8ddbecf109748bb69e341f - removed suppress warning 'unused' that caused a warning because it was

- [Ariel Kolikant] 68b72bc80e8e1f13776e8ec093729f9099e4b1ad - #305 opens page for adding

- [Ariel Kolikant] 304a108fd641c603947a38043845ce754b827fc1 - tells me i have changes

- [Alexander Kaplan] fc611583c7e222e8e88687f0f9818c2bb60d32b0 - fix tests failing for database



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/576/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
